plugins/woocommerce/includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -690,6 +690,10 @@ public static function get_updates_count() {
 				continue;
 			}
 
+			if ( ! is_plugin_active( $plugin['_filename'] ) ) {
+				continue;
+			}
```

Comment:
This is a change that I need to double check with the team. This will **not** count deactivated products in the update badge.